openquake/risklib/valid.py

```diff
@@ -835,6 +835,18 @@ def positiveints(value):
     return ints
 
 
+def weights(value):
+    """
+    >>> weights('0.4 0.3')
```

Comment:
We are already using the conventions that the weights are numbers from 0.0 to 1.0 in the logic tree files, so I want to keep compatibility with the past.